LOAN/LoanTypes/SecuredLoans.rdf

```diff
@@ -71,14 +71,14 @@
 		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-sec;SecuredLoan"/>
```

Comment:
These seem orthogonal. Can someone not guarantee a loan without it also being secured? See https://www.finder.com/guaranteed-loan (3rd heading).

Note also that a guarantee may be *limited* to only part of the amount of the loan.

Reply:
merged a bit more with LoanCore, which also defined secured loan, but may not have completely addressed this ... should raise another issue to merge this, and unsecured loans into LoanCore